ide/editor.completion/src/org/netbeans/modules/editor/completion/PatchedHtmlRenderer.java

```diff
@@ -176,7 +176,7 @@ private static double _renderPlainString(
                 }
 
                 double chWidth = wid / chars.length;
-                int estCharsToPaint = new Double(w / chWidth).intValue();
+                int estCharsToPaint = Double.valueOf(w / chWidth).intValue();
```

Comment:
Continued from `CreateTableDialog.java`, line 474:

```suggestion
                int estCharsToPaint = (int) (w / chWidth);
```

This pattern is found multiple times in this PR.

Reply:
Hey @matthiasblaesing , I would like to leave this one "as is" if you are okay with it. The reason is the Java Lang. Spec. specifices narrowing behavior of intValue() and in a different section it talks about casting. It's not really clear that they are equivalent. I agree it can be made "cleaner". But, I decided to just go with "guaranteed" behavior here.

Reply:
Ok - keep it, but from my perspective, if ` Double.valueOf(<double_value>).intValue() == ((int) <double_value>)` is not true for all instances of `<double_value>`, then the language is broken in its core. My reasoning:

- `Double.valueOf` should yield the same value as autoboxing
- `Number#intValue` should in my mind yield the same value as unboxing the the corresponing primitive box and casting it to `int`

I don't see a different possible interpretation.

Reply:
`valueOf(..).fooValue()` can be shortened to the more efficient/compact `parseFoo(..)`. NetBeans should actually give this suggestion already as a hint.

Reply:
oh wait, ignore that. This isn't parsing anything. Nevermind.

Reply:
this is int divided by double and the result (double) is casted down to int.
please change to
`(int) (w / chWidth)`
which is the equivalent

Double#intValue is implemented as:
```java
    public int intValue() {
        return (int)value;
    }
```
so we are essentially inlining a method here.

Reply:
You write agreed, but it is not done. Please recheck (the same is true for the other locations in this file).